### PR TITLE
Automate ovirt gluster network post hc deployment

### PR DESCRIPTION
**Pull request by gobindadas**
RHBZ: 1835574

### CODE REVIEW

File: playbooks/hc-ansible-deployment/tasks/add_hosts_storage_domains.yml [hunk 1]
```diff
@@ -47,3 +47,16 @@
...
+    ovirt_network:
+      data_center: "{{ datacenter_name }}"
+      name: gluster_net
+      vm_network: false
+      auth: "{{ ovirt_auth }}"
+      clusters:
+        - name: "{{ cluster_name }}"
+          assigned: true
+          required: false
+          gluster: true
```
Comment (satheesaran) [inline playbooks/hc-ansible-deployment/tasks/add_hosts_storage_domains.yml:62]: This network is also a migration network, please add migration: true

**Comment by satheesaran** [comment]
@gobindadas This part of the code creates the gluster logical network, but this doesn't attach that network to the hosts. ovirt_host_network ansible module helps in attaching the network to the required host network interface.

Now you need to get that interface as input from the user for each host. so it should be part of gluster_inventory.yml

Please make that change too

File: playbooks/hc-ansible-deployment/tasks/add_hosts_storage_domains.yml (hunk 1 above)
Comment (satheesaran) [reply playbooks/hc-ansible-deployment/tasks/add_hosts_storage_domains.yml:62]: We need to ignore this patch as we decided to create a separate playbook for gluster_network creation, as the hosts are added to the cluster in the async fashion, creation of network had to wait for long time which also prolongs the deployment

**Comment by gobindadas** [comment]
Verified, it's working as expected.
Please review @satheesaran @pkesavap

**Pull request merged**